.github/workflows/example.yml

```diff
@@ -44,10 +51,11 @@ jobs:
           stack-arguments: --resolver ${{ matrix.resolver }}
           cache-prefix: ${{ matrix.resolver }}/
 
-      - if: ${{ runner.os != 'windows' }}
+      - if:
+        shell: bash
         run: |
-          [[ "${{ steps.stack.outputs.compiler }}" = "ghc-8.10.4" ]]
-          [[ "${{ steps.stack.outputs.compiler-version }}" = "8.10.4" ]]
```

Comment:
How the heck was this passing before?

Reply:
You didn't pass `stack-arguments` that set the resolver

https://github.com/freckle/stack-action/blob/ac0dd8dbe5850e6597a5a4ce5fe4e06613f490ce/.github/workflows/example.yml#L44

https://github.com/freckle/stack-action/blob/ac0dd8dbe5850e6597a5a4ce5fe4e06613f490ce/action.yml#L177-L179
 
So, the action always used [lts-18.3](https://www.stackage.org/lts-18.3) with `ghc-8.10.4`

https://github.com/freckle/stack-action/blob/ac0dd8dbe5850e6597a5a4ce5fe4e06613f490ce/example/stack.yaml#L1

and set this version as an input

https://github.com/freckle/stack-action/blob/ac0dd8dbe5850e6597a5a4ce5fe4e06613f490ce/action.yml#L182